kitchen-tests/Gemfile.lock

```diff
@@ -59,9 +59,8 @@ GEM
     excon (0.51.0)
     faraday (0.9.2)
       multipart-post (>= 1.2, < 3)
-    ffi (1.9.14)
```

Comment:
yup. guess we need to look into why that is getting yanked but will add it manually now